src/app/static/src/app/components/plots/table/DifferenceColumnRenderer.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+    <span class="d-inline-flex">
+        <!--Setting a transform to push icon up 1 pixel to align slightly
```

Comment:
I was fighting with this a bit, not sure what it looks like on your guys end? Just looks like to me the actual character ▲has more white space above than below and so is rendering kind of ugly on firefox. Couldn't really work out a nice way to make it align.

Actually looking at the screenshots now it looks like from chromium perhaps I didn't need to push it up a pixel and that chromium is displaying it nicely.

I tried looking to see if there was an appropriate vue-feather or font-awesome icon I could use instead which might be easier to centre. But vue-feathers was a big rounded triangle so not quite right and all the similar looking ones on font-awesome were premium.

Also considered just drawing svg path in manually but that seemed like madness. 

Curious on your thoughts and how you would solve this?

Reply:
spacing looks ok to me on both Firefox and Chrome; on my end I am seeing the same icon size in both browsers, the larger size that Mantra showed in the second screenshot

Reply:
If consistent sizing is an issue and you do end up deciding to replace with an icon, looks like the "play" icon from vue-feather might be suitable. Could do a 90 degree rotation in either direction to get it oriented correctly.

Reply:
Thanks! Rotated play icon is a great idea. If this looks ok to you both then let's go with it, if we see some issues down the line we can switch to the play button icon. I don't mind the difference in size on linux from chrome to firefox, as long as they are centered :D